### DISCUSSION TITLE
Is it possible to pass custom arguments into a custom prisma migration script?

### DISCUSSION THREAD

**Discussion by konrad-peeriq** [prisma/prisma#17918] [Q&A] [answered]
Is it possible to pass custom arguments into a custom prisma migration script?

Use case: use a migration script to create a new database user, pass the password into the migration script (environment variable)

An example of this: Flyway has placeholders which allow you to pass custom environment variables into migration scripts

> Flyway supports [placeholders](https://flywaydb.org/documentation/configuration/parameters/placeholders) that can be used to pass custom values into sql scripts
> 
> `flyway -placeholders.db_user=${DB_USER} -placeholders.db_user_pw=${DB_USER_PASSWORD} ...`
>
> You can reference these placeholders in SQL like this:
> 
> `CREATE ROLE ${db_user} LOGIN PASSWORD '${db_user_pw}';`
> 
> DB_USER and DB_USER_PASSWORD are environment variables and should be wired during the deployment to get values from AWS SSM using deployment tool capabilities.
> https://stackoverflow.com/a/70653479

**Comment by nurul3101** [comment, marked as answer]
Hello @konrad-peeriq 👋 

I don't think it's possible yet. I believe we don't have a feature request for this kind of use case well. Can you create a [Feature Request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that our team can consider implementing this use case in future?

**Reply by konrad-peeriq** [reply]
Hi @nurul3101!

Thanks so much for the very quick and helpful reply! Added the feature request: https://github.com/prisma/prisma/issues/17924

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
